src/styles/index.scss

```diff
@@ -1,3 +1,199 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
```

Comment:
The `display: flex;` property should be placed before `flex-direction: column;`, `justify-content: center;` and `align-items: center;` properties. This is a common practice to first define a flex container and then specify its properties.